core/cloudflow-akka/src/main/resources/akka-cluster-local.conf

```diff
@@ -0,0 +1,23 @@
+akka {
+  actor {
+    provider = "cluster"
+  }
+
+  discovery {
+    method = akka-dns
+  }
+
+  cluster {
+    min-nr-of-members = 1
+    log-info = on
+    shutdown-after-unsuccessful-join-seed-nodes = 60s
+
+    seed-nodes = ["akka.tcp://akka_streamlet@"${akka.remote.netty.tcp.hostname}":"${akka.remote.netty.tcp.port}]
+  }
+
+  remote {
+    netty.tcp {
+      hostname = "127.0.0.1"
+    }
+  }
+}
```

Comment:
@RayRoestenburg do we need to make some of the stuff configurable through external configuration ? Possibly when we have our configuration system in place .. I don't have much fu on akka cluster but if there are other things that the user may want to configure with akka cluster how will they do that ?

Reply:
Users can put it in their own reference.conf file, or application.conf file for now, for overrides. And yes the config system will make this a lot easier. We'll start work on that soon.

Reply:
But for now this is fine.